configs/imx8mm_evk_defconfig

```diff
@@ -139,3 +139,4 @@ CONFIG_DM_VIDEO=y
 CONFIG_VIDEO_LCD_RAYDIUM_RM67191=y
 CONFIG_VIDEO_ADV7535=y
 CONFIG_SYS_WHITE_ON_BLACK=y
+CONFIG_NET_RANDOM_ETHADDR=y
```

Comment:
This is not described in the commit message... Why this config option is needed here?